src/ui/public/visualize/visualize.less

```diff
@@ -5,7 +5,6 @@ visualization {
   flex-direction: column;
   height: auto;
   width: 100%;
-  overflow: auto;
```

Comment:
This should probably be checked with kibana vis team.
When `overflow: auto`, if the tooltip is high, it crosses the panel dimensions and gets cut.
Without `overflow`, the tooltip simply goes beyond the panel (Which is fine I guess).
I didn't find any other issues with it

Reply:
It's probably best to leave this change out. EUI has a new tooltip component which should be (if not already) an easy fit into TSVB.

@simianhacker @mattapperson does TSVB use EUI tooltip yet, or is it in the near future?